Doc/reference/compound_stmts.rst

```diff
@@ -411,7 +411,8 @@ This allows common :keyword:`try`...\ :keyword:`except`...\ :keyword:`finally`
 usage patterns to be encapsulated for convenient reuse.
 
 .. productionlist:: python-grammar
-   with_stmt: "with" `with_item` ("," `with_item`)* ":" `suite`
+   with_stmt: ( "(" `with_stmt_contents` ")" | `with_stmt_contents` ) ":" `suite`
+   with_stmt_contents: "with" `with_item` ("," `with_item`)*
```

Comment:
The grammar here looks wrong, right? Shouldn't it be like this?
```suggestion
   with_stmt: "with" ( "(" `with_stmt_contents` ","? ")" | `with_stmt_contents` ) ":" `suite`
   with_stmt_contents: `with_item` ("," `with_item`)*
```

Reply:
Indeed, this was a copy-paste error when breaking the statement into 2. Thanks for the catch!